### ISSUE TITLE
Combulator expansion

### ISSUE THREAD

**Issue by mkruselj** [surge-synthesizer/surge#3811]
Once we have more FX params to play with, it would be great to add:

- [ ] Input gain (-inf to 0 dB, so basically `ct_amplitude`)
- [ ] Noise type (menu, white, pink, blue, [grey](https://en.wikipedia.org/wiki/Colors_of_noise#Grey_noise), dust)
- [ ] Individual feedback per comb?

**Comment by magnetophon** [comment]
Offset mode for the 2nd and 3rd frequency?

How about a slider that crossfades between the various noise types instead of a menu?

**Comment by baconpaul** [comment]
The 2ns and 3rd frequency are already offsets fwiw

**Comment by magnetophon** [comment]
Ah, sorry, I didn't update to the version with Combulator yet.
In any case, it would be nice to have both options: absolute and relative.

**Comment by mkruselj** [comment]
The resonance (feedback) is already bipolar.

Bipolar mix didn't work properly or as expected - it bled through dry input at -100% wet. You don't really want that :)

**Comment by magnetophon** [comment]
>  it bled through dry input at -100% wet.

How come?
Sounds like an implementation bug, right?
Nothing to do with the concept itself, or?

**Comment by baconpaul** [comment]
> Sounds like an implementation bug, right?

yeah i thought we fixed it but now i see we fixed it by setting the lower bound to 0! Will take a look
